docs/custom-flows/passkeys.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Build a custom authentication flow using passkeys
+description: Learn how to allow your users authenticate with passkeys with Clerk.
+---
+
+# Build a custom authentication flow using passkeys
+
+<Callout type="warning">
+This guide is for users who want to build a custom user interface using the Clerk API. To authenticate users with passkeys using a prebuilt UI, you should use Clerk's [`<SignIn />`](/docs/components/authentication/sign-in) component. See [the sign-in and sign-out options docs](/docs/authentication/configuration/sign-up-sign-in-options) to learn more.
+</Callout>
+
+Clerk supports passwordless authentication via passkeys, enabling users to sign in without having to remember a password. Instead, users select a passkey associated with their device, which they can use to authenticate themselves.
+
+This guide will teach you how to build custom flows for creating, signing users in with, and managing passkeys.
+
+## Create user passkeys
+
+To create a passkey for a user, you must call the [`createPasskey()`](/docs/references/javascript/user/user#create-passkey) method on the [`User`](/docs/references/javascript/user/user) object as shown in the following example:
```

Comment:
what do you think about shortening this - the user will still be taken to the `User` docs, and will still get an example of how to access and use `User`.

```suggestion
To create a passkey for a user, you must call [`User.createPasskey()`](/docs/references/javascript/user/user#create-passkey) as shown in the following example:
```